modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala

```diff
@@ -86,9 +86,9 @@ object GmosControllerEpics extends GmosSouthController {
     case s            => DC.setShutterState(encode(s))
   }
 
-  private def roiNumUsed(s: RegionsOfInterest): Int = s match {
-    case RegionsOfInterest(\/-(rois)) => rois.length
-    case RegionsOfInterest(-\/(b))    => 1
+  private def roiNumUsed(s: RegionsOfInterest): Int = s.rois match {
```

Comment:
This change maybe an error on the compiler. The previous match produced a warning on `non-exhaustive` matching checks